src/metabase/events/recent_views.clj

```diff
@@ -3,44 +3,26 @@
   (:require
    [metabase.api.common :as api]
    [metabase.events :as events]
+   [metabase.models.audit-log :as audit-log]
    [metabase.models.recent-views :as recent-views]
+   [metabase.util :as u]
    [metabase.util.log :as log]
    [methodical.core :as m]))
 
 (derive ::event :metabase/event)
 
 (derive :event/dashboard-read ::event)
 (derive :event/table-read ::event)
+(derive :event/card-read ::event)
```

Comment:
Changed `recent_views` to only be triggered by API-level read events, not by card queries. `recent_views` should essentially mirror the same events we're logging in `view_log`, but only contain the most recent N (= 30) views per user so that it's quicker to access for non-audit queries